public/styles.css

```diff
@@ -110,6 +110,11 @@ nav li p {
   margin: 0;
 }
 
+nav li p.small {
```

Comment:
I appreciate you noticing the overflowing text and wanting to do something about it. I think there are still some alignment issues and it does look a bit odd with the text only being smaller on one entry. I'd rather not spend a lot of time going back and forth trying to tweak values to make it look right, only to later find out that it looks wrong on different devices because of font metrics.

I realise this is just delaying a proper solution until something with an _even longer_ name gets added, but there is enough space in the header to up the size of the `nav li` to `6em` and the SVGs to `3em` to match. That gives _just_ enough space for the ChromeOS text to fit, at least in Segoe UI and Arial (Windows' sans-serif fallback) which is all I can test with. 

|Small text|Larger items|
|---|---|
|![image](https://github.com/s-thom/howtoscreenshot/assets/7552536/d8917f6f-f0cb-4bca-9ef2-5b9ddee747f8)|![image](https://github.com/s-thom/howtoscreenshot/assets/7552536/ad1b7e69-71b3-4899-9fee-e136e89faba7)|

I'm open to other ideas too if you can think of anything else.